lib/core/auth.js

```diff
@@ -234,6 +229,41 @@ export default class Auth {
       })
   }
 
+  requestApollo (operation) {
+    const isQuery = operation.hasOwnProperty('query')
+    const opFn = this.ctx.app.apolloProvider.defaultClient[
+      isQuery ? 'query' : 'mutate'
+    ]
+    const res = opFn(operation)
+      .then(response => {
+        if (operation.propertyName) {
+          const propVal = getProp(response.data, operation.propertyName)
+          return propVal
+        } else {
+          return response.data
+        }
+      })
+      .catch(error => {
+        // Call all error handlers
+        this.callOnError(error, { method: 'request' })
+
+        // Throw error
+        return Promise.reject(error)
+      })
+    return res
+  }
+
+  request (endpoint, defaults) {
+    const et = _.merge({}, defaults, endpoint)
+    const _endpoint = typeof defaults === 'object' ? et : endpoint
+
+    if (this.strategy.options.useApollo) {
```

Comment:
General tip: We shouldn't assume anything about strategy options. They may internally parse and handle options.

Reply:
I went forward on the assumption that graphQL is growing fast, and apollo is the clear frontrunner.

In hindsight if we keep Apollo in the core (major quality of life for Apollo devs), then this should be re-named to isGraphql or something more agnostic